test/KHI_growthRate/bin/ci.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+#
+# This file is part of PIConGPU.
+# Copyright 2022 PIConGPU contributors
+# Authors: Mika Soren Voss
+# License: GPLv3+
+#
+
+currentPath=$(cd `dirname $0` && pwd)
+create="true"
+
+while [[ $# > 0 ]] ; do
+        case "$1" in
+
+                -t|--testPar)
+                        testPar="$2"
+                        shift
+                        ;;
+
+                -s|--simDir)
+                        simDir="$2"
+                        shift
+                        ;;
+
+                -f|--force)
+                        force="true"
+                        ;;
+
+                -c|--create)
+                        create="false"
+                        ;;
+
+                --help|-h)
+                        echo "With this file the test of the growth rate of the KHI can"
+                        echo " be started. The specified test case is used by default "
+                        echo "(see parameters)."
+                        echo "Alternatives to run the test suite are validate.sh and MainTest.py."
+                        echo "If the parameter -t is not set, it is assumed that"
+                        echo " picongpu is in the home directory"
+                        echo "Usage:"
+                        echo "  --testPar|-t \"direction/to/picongpu/\""
+                        echo "  --simDir|-s \"direction/where/simulation/should/be/build/\""
+                        echo "  -c|--create do not run pic-create"
+                        echo "  -f|--force  - Override if 'destinationPath' exists."
+                        echo "  --help|-h"
+                        echo "Example:"
+                        echo " bash ci.sh - creates a new temporary folder in which the "
+                        echo "              test-suite is run and then deletes it"
+                        exit 0
+                        ;;
+        esac
+        shift
+done
+
+if [ -z "$simDir" ] ; then
+        date_stamp=$(date +"%F-%h-%M-%S")
+        simDir=/tmp/$date_stamp
+fi
+
+if [ -d "$simDir" ] ; then
+    if [ "$force" == "true" ] ; then
+        echo "Warning: using existing folder on user-request [-f]"
+    else
+        echo "Destination path already in use, cannot create new folder" >&2
+        exit 1
+    fi
+fi
+
+if [ -z "$testPar" ] ; then
+        testPar=$HOME
+fi
+
+source $testPar/picongpu/etc/picongpu/bash-devServer-hzdr/gpu_picongpu.profile.example
```

Comment:
please remove this line. Whoever is executing this script must set up their environment before.
The CI is not running on the local dev server.

Reply:
You can assume that `pic-create` is available.